cdk/lib/cdk-playground.ts

```diff
@@ -43,12 +59,17 @@ export class CdkPlayground extends GuStack {
 				minimumInstances: 1,
 				maximumInstances: 2,
 			},
-			imageRecipe: 'arm64-bionic-java11-deploy-infrastructure',
+			imageRecipe: 'jacob-test',
```

Comment:
This recipe included the [OTEL Collector AMIgo role](https://github.com/guardian/amigo/tree/main/roles/otel-collector-arm).